Docs: clarify where field configs are set in node graph visualization (#121245)
grafana/grafana: docs/sources/visualizations/panels-visualizations/visualizations/node-graph/index.md

| nodeRadius    | number        | Radius value in pixels. Used to manage node size.                                                                                                                                                                                                                                                                                                                        |
| highlighted   | boolean       | Sets whether the node should be highlighted. Useful for example to represent a specific path in the graph by highlighting several nodes and edges. Default: `false`                                                                                                                                                                                                      |

### Field config

The tables above reference `config.displayName`, `config.color.fixedColor`, and `field.config.color.mode`. These are set in the `config` property of each field in the data frame response from your data source.

> If you are building a data source plugin or returning data frames from an API, set field configs in the field definition. For example, in a JSON data frame response:

Author: Addressed in `fed86eb2`. I rewrote the section heading and surrounding prose to use `field configuration` rather than `config/configs` in the running text.

```json
{
  "fields": [
    {
      "name": "arc__success",
      "type": "number",
      "values": [0.7],
      "config": {
        "displayName": "Success rate",
        "color": {
          "fixedColor": "green",
          "mode": "fixed"
        }
      }
    },
    {
      "name": "detail__latency",
      "type": "number",
      "values": [42],
      "config": {
        "displayName": "Latency (ms)",
        "unit": "ms"
      }
    }
  ]
}
```

> Common field config properties used by the node graph panel:

Author: Addressed in `fed86eb2`. This wording now uses `field configuration` consistently in the explanatory text.

| Property                  | Description                                                                                                  |
| ------------------------- | ------------------------------------------------------------------------------------------------------------ |
| `config.displayName`      | Human-readable label shown in tooltips and context menus instead of the raw field name.                       |

> | `config.color.fixedColor` | Fixed color for `arc__*` fields (for example, `"green"`, `"red"`, `"#FF5733"`).                              |

Author: Addressed in `fed86eb2`. I updated the phrasing in this section to avoid the Vale hit on `config`.

| nodeRadius    | number        | Radius value in pixels. Used to manage node size.                                                                                                                                                                                                                                                                                                                        |
| highlighted   | boolean       | Sets whether the node should be highlighted. Useful for example to represent a specific path in the graph by highlighting several nodes and edges. Default: `false`                                                                                                                                                                                                      |

### Set field options

> The tables above reference `config.displayName`, `config.color.fixedColor`, and `field.config.color.mode`. Set these values in each field's `config` object in the data frame returned by your data source.

Reviewer: ```suggestion
The tables above reference `config.displayName`, `config.color.fixedColor`, and `field.config.color.mode`. Set these values in each field's `config` object when you build the data frame in your data source.
```

> If you are building a data source plugin or returning data frames from an API, define these field options in the field definition. For example, in a JSON data frame response:

Reviewer: ```suggestion
The following example shows the field definitions for a node graph data frame, with these options set on the `config` object of each field.
```
